Scripts/new_feature.sh

```
@@ -7,6 +7,14 @@ if [ -z "$FEATURE_NAME" ]; then
   exit 1
 fi
 
+# Confirm with user
+echo "Creating feature: $FEATURE_NAME"
+read -p "Is this correct? (y/n): " confirm
```

Shellcheck (0.11.0)

[warning] In POSIX sh, read -p is undefined.

(SC3045)